pkg/localmetrics/localmetrics.go

```diff
@@ -193,3 +202,19 @@ func IncrementLetsEncryptMaintenanceErrorCount() {
 func IncrementDnsErrorCount() {
 	MetricDnsErrorCount.Inc()
 }
+
+// register missingCertificates and certificateRetrievalErrors metrics in the prometheus
+func init() {
+	metrics.Registry.MustRegister(missingCertificates)
+	metrics.Registry.MustRegister(certificateRetrievalErrors)
+}
```

Comment:
Is there a reason we're initializing these two metrics here? How are the other metrics initialized? Should we try to initialize them the same way as the others?

Reply:
@iamkirkbater  i wasn't able to find anyway they are initializing these metrics the current way therefore, i initalized these new metrics the new way. However, ill be reverting back the new metrics changes once I find the root cause on why current metrics are not getting all the certs inc hibernating, error and LS clusters.